### PR TITLE
[CWS] Do not exclude argv[0] from args

### PR DESCRIPTION
**Pull request by safchain**
### What does this PR do?

Parse argv[0] to avoid envs being parsed as args in case of argc == 0.

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] The appropriate `team/..` label has been applied, if known.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

File: docs/cloud-workload-security/agent_expressions.md [hunk 1]
```diff
@@ -133,6 +133,7 @@ The *file.rights* attribute can now be used in addition to *file.mode*. *file.mo
 | -------- | ---- | ---------- |
 | `container.id` | string | ID of the container |
 | `container.tags` | string | Tags of the container |
+| `process.ancestors.arg0` | string |  |
```
Comment (sarina-dd) [inline docs/cloud-workload-security/agent_expressions.md:136]: @safchain 👋 Thanks for opening this PR! It looks like the descriptions are empty here. Do you plan on adding descriptions?

Comment (safchain) [reply docs/cloud-workload-security/agent_expressions.md:136]: @sarina-dd thanks, my bad I forgot to commit a file, I updated the PR

File: releasenotes/notes/cws-expose-arg0-db85ae10cdeafd34.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Each section from every release note are combined when the
```
Comment (lebauce) [inline releasenotes/notes/cws-expose-arg0-db85ae10cdeafd34.yaml:1]: nit: remove these comments

Comment (safchain) [reply releasenotes/notes/cws-expose-arg0-db85ae10cdeafd34.yaml]: Done

**Review by sarina-dd — APPROVED** [review]

**Review by lebauce — APPROVED** [review]

**Pull request merged**